website/docs/r/app_installation_repositories.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "github"
+page_title: "GitHub: github_app_installation_repository"
+description: |-
+  Manages the associations between app installations and repositories.
+---
+
+# github_app_installation_repositories
+
+~> **Note**: This resource is not compatible with the GitHub App Installation authentication method.
+
+This resource manages relationships between app installations and repositories
+in your GitHub organization.
+
+Creating this resource installs a particular app on multiple repositories.
+
+The app installation and the repositories must all belong to the same
+organization on GitHub. Note: you can review your organization's installations
+by the following the instructions at this
+[link](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/reviewing-your-organizations-installed-integrations).
+
+## Example Usage
+
+```hcl
+# Create some repositories.
+resource "github_repository" "some_repo" {
+  name = "some-repo"
+}
+
+resource "github_repository" "another_repo" {
+  name = "another-repo"
+}
+
+resource "github_app_installation_repositories" "some_app_repos" {
+  # The installation id of the app (in the organization).
+  installation_id          = "1234567"
+  selected_repository_ids  = [github_repository.some_repo.name, github_repository.another_repo.name]"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `installation_id`       - (Required) The GitHub app installation id.
+* `selected_repositories` - (Required) A list of repository names to install the app on.
+
+~> **Note**: Due to GitHub API limitations, deleting this resource will leave one repository with the app installed. Manually uninstall the app after deleting the resource.
```

Comment:
How is the remaining repository determined? Is it deterministic?

Reply:
See comment below, I'll update the docs to reflect

Reply:
@kfcampbell 
Re running the tests: I just pulled my branch and made sure to set `GITHUB_TOKEN`, `GITHUB_ORGANISATION`, and `APP_INSTALLATION_ID` and it ran fine.
I guess recheck your `APP_INSTALLATION_ID` is the correct one for your org specified in `GITHUB_ORGANISATION`? I was able to recreate your error when I deliberately put in an incorrect installation ID...

Reply:
Oh my gosh you're totally right! Thanks for the help troubleshooting the testing.